docs/components/tools.md

```diff
@@ -811,7 +814,7 @@ Flags:
       --rewrite.to-relabel-config-file=<file-path>
                                 Path to YAML file that contains relabel configs
                                 that will be applied to blocks
-      --tmp.dir="/tmp/thanos-rewrite"
+      --tmp.dir="/var/folders/pr/h0912y1s3zj507j30fzj87gh0000gn/T/thanos-rewrite"
```

Comment:
What's this random directory? :thinking:

Reply:
I'm not sure actually, this change was generated when I ran `make check-docs`, but the docs check kept breaking. I removed this change and the docs check passed, I didn't understand why since it also rans `mack check-docs` 🤔

Reply:
This thing happens to me. I feel it is an issue specific to MacOS. The temp dir is generated incorrectly.